### ISSUE TITLE
There should be a bug in vikit_common cmakelists

### ISSUE THREAD

**Issue by jian-li** [uzh-rpg/rpg_vikit#31]
The line 95 should be modified to *.h**, otherwise if make install the .hpp file won't be installed!

**Comment by taskx6004** [comment]
Yes, I have tested this and found i have to cp the .hpp to /usr/local/include if i build the svo project without ROS.
